src/main/java/org/tron/core/capsule/AccountCapsule.java

```diff
@@ -522,6 +522,10 @@ public long getStorageUsage() {
     return this.account.getAccountResource().getStorageUsage();
   }
 
+  public long getStorageLeft() {
+    return getStorageUsage() - getStorageUsage();
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Correct one of the identical sub-expressions on both sides of operator "-" ([squid:S1764](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS1764))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1217&issues=AWT0NMSEvlEfaeH0ZGhK&open=AWT0NMSEvlEfaeH0ZGhK)

[ISSUE_KEY]: AWT0NMSEvlEfaeH0ZGhK